dashboard/client/src/service/node.ts

```diff
@@ -1,10 +1,18 @@
-import { NodeDetailRsp, NodeListRsp } from "../type/node";
+import {
+  NodeDetailRsp,
+  NodeListRsp,
+  NodeWithWorkersListRsp,
+} from "../type/node";
 import { get } from "./requestHandlers";
 
 export const getNodeList = async () => {
   return await get<NodeListRsp>("nodes?view=summary");
 };
 
+export const getNodesWithWorkers = async () => {
+  return await get<NodeWithWorkersListRsp>("nodes?view=details");
```

Comment:
Can we think more about the large scale scenes here? If we deploy the Ray cluster in a large scale, we will fetch a big detail table by this API. So, can we make it to be interactive? When the users click a node, we fetch the details for only one node on demand.